app/models/team_performance.rb

```diff
@@ -0,0 +1,64 @@
+class TeamPerformance
+# Memo: calculates Team's Performance Score for Supervisor's Dashboard
+
+  def initialize(team)
+    @team = team
+  end
+
+  def score
+    @score = 0
+
+    if @team.comments.any?
+      comments_score
+    elsif Time.now-2.days > Season.current.starts_at && Time.now < Season.current.ends_at
+      # Maybe we should change the +3 score temporarily for the upcoming week,
+      # because before Wed 16th comments-without-text were not being saved.
+      @score += 3
+    end
+
+    if @team.activities.any?
+      activity_score
+    elsif Time.now-2.days > Season.current.starts_at && Time.now < Season.current.ends_at
+      @score += 3
+    end
+    evaluate_performance
```

Comment:
To me, `score` suggests a numerical return value but instead it returns a symbol. Does it stress you to tear the score-calculation and the semantical evaluation apart? The method `evaluate_performance` could be renamed to `performance` and return the symbol.